lib/interface_quda.cpp

```diff
@@ -3826,43 +3790,37 @@ void invertMultiShiftQuda(void **hp_x, void *hp_b, QudaInvertParam *param)
   // cache is written out even if a long benchmarking job gets interrupted
   saveTuneCache();
 
-  profileMulti.TPSTOP(QUDA_PROFILE_TOTAL);
-
   profilerStop(__func__);
+  popProfile();
 }
 
 void computeKSLinkQuda(void *fatlink, void *longlink, void *ulink, void *inlink, double *path_coeff, QudaGaugeParam *param)
 {
-  profileFatLink.TPSTART(QUDA_PROFILE_TOTAL);
-  profileFatLink.TPSTART(QUDA_PROFILE_INIT);
```

Comment:
Why was profiling `QUDA_PROFILE_INIT` deleted here?

Reply:
This was intentional.  Profiling of memory allocations is now done automatically inside `quda_ptr`.